server/test/createFile_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2018 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package test
+
+import (
+	"fmt"
+	"github.com/moov-io/ach/server"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/moov-io/ach"
+)
+
+// TestCreateFile Tests Creating an ACH File From JSON
+// These tests are not designed to have a failure but can be used as TEST for JSON formatting for an SEC Code
+func TestCreateFile(t *testing.T) {
+	createFileError := func(path string, msg string) {
+		_, err := os.Open(path)
```

Comment:
What if instead we read the existing `*.ach` files in each `test/*-read/` directory, convert those to JSON in memory and then check? We wouldn't need to keep the JSON files in this repository or up to date in future changes.

Reply:
Sure we can do that.  I was thinking we would want to I’ve users sample json files for each SEC code.

Reply:
Im not sure the point of this issue then because we know we support creating files from json.

Reply:
Im not sure the point of this issue then because we know we support creating files from json.

Reply:
If we want the JSON files to live in the repo then can we generate them as part of `make` and put them in an `examples/json/` directory.

Reply:
I certainly am ok doing it.  I just want to make sure, we have the examples.

Reply:
In the interim I'll look at putting this in the test\server-example directory or the examples directory.  From there I can automate it...

Reply:
I'd like it to be automated as much as possible that way we don't have to maintain all these JSON files going forward. Having a unit test write them to disk is fine, but could introduce noisy diffs in the future.

Reply:
Ok got it.